.gcloudignore

```diff
@@ -1,2 +1,6 @@
 #!include:.gitignore
 #!include:.dockerignore
+
+# Copy cached packages to prevent from
+# fetching them from registry
+!.yarn/cache
```

Comment:
This might be slower though if the pipe between the client and gcloud service is slower/smaller than gcloud's internet access. But let's keep it in for now.

Reply:
Swiss internet :P - changing, also bad regarding cache invalidations